pkgs/development/libraries/boost/generic.nix

```
@@ -71,6 +71,8 @@ let
     "link=${link}"
     "-sEXPAT_INCLUDE=${expat.dev}/include"
     "-sEXPAT_LIBPATH=${expat.out}/lib"
+  ] ++ optionals (versionAtLeast version "1.70") [
```

If  1.71 doesn't need this, then, would it make sense to check for equality instead?
```suggestion
  ] ++ optionals (version == "1.70") [
```

---

I was considering this, because I feared some mass rebuild, which turns out isn't an issue. I think it's more explicit and it doesn't rely on an implementation detail of the build system that might change.